src/include/ASTree.h

```diff
@@ -237,4 +238,9 @@ struct FnCall : public Expr {
         , m_args(std::move(args)) { }
 };
 
+struct ExprStmt : public Stmt {
```

Comment:
Does this really need to be its own `struct`? I was intending for a plain `Expr` to represent both an expression and an expression statement.

Reply:
I think it's easiest if we have a separate struct since we want to visit it differently if it's an expr vs exprstmt. (For expr we need to leave a value on the stack, not for exprstmt.)

Reply:
Okay, fair enough.